qurator/eynollah/processor.py

```diff
@@ -58,9 +64,14 @@ def process(self):
                 'override_dpi': self.parameter['dpi'],
                 'logger': LOG,
                 'pcgts': pcgts,
-                'image_filename': str(image_filename)
+                'image_filename': page.imageFilename,
+                'image_pil': page_image
                 }
-            Eynollah(**eynollah_kwargs).run()
+            eynollah = Eynollah(**eynollah_kwargs)
+            if models is not None:
+                # reuse loaded models from previous page
+                eynollah.models = models
+            pcgts = eynollah.run()
```

Comment:
But we'll still need to assign `models = eynollah.models` after the `run`, right?

Reply:
61bcb43

Reply:
> But we'll still need to assign `models = eynollah.models` after the `run`, right?

I don't think so. AFAIK the model instantiation will always be the same. But I could be wrong. Also, I fail to see any risk in going back and forth between Eynollah and EynollahProcessor.

Reply:
Ah, only now I get the question. It seems I did completely forget to save the loaded models from Eynollah to EynollahProcessor in the first place. So :+1: for your addition.